client/src/App.tsx

```diff
@@ -5,9 +5,9 @@ function App() {
   return (
     <div>
       <Header />
-      <div className="container mx-auto pt-28">
+      <main className="container mx-auto pt-28 min-h-screen">
```

Comment:
```suggestion
      <main className="container mx-auto pt-28 min-h-screen min-w-full bg-background-100">
```
이렇게 하면 배경화면 컬러도 한번에 지정 가능할 것 같아요
이거까지 해주시면 감사하겠습니다 👍

Reply:
`width`은 `container`에서 지정해줘서 설정할 필요없어요!
배경 반영했습니다~

Reply:
제 [커밋](https://github.com/softeerbootcamp-3rd/Team9-HexaCore/pull/19/commits/3f5dd10cd56af447fb23633af99297934463d232) 확인해 보시겠어요?

Reply:
맨 위에다 하면 되는군요 확인했습니당